lib/screens/login.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+import 'package:admu_student_app/screens/onboarding.dart';
+
+class login extends StatelessWidget {
```

Comment:
class names usually start with capital and use camelcase: `class Login extends...` or `class BottomNavBar extends ...`